base/univention-config-registry/python/univention/config_registry/frontend.py

```diff
@@ -140,13 +140,14 @@ def handler_set(args, opts=dict(), quiet=False):
 			if sep_set == -1 and sep_def == -1:
 				print("W: Missing value for config registry variable '%s'" % (arg,), file=sys.stderr)
 				continue
-			else:
-				if sep_set > 0 and sep_def == -1:
-					sep = sep_set
-				elif sep_def > 0 and sep_set == -1:
-					sep = sep_def
-				else:
-					sep = min(sep_set, sep_def)
+
+                        if sep_set > 0 and sep_def == -1:
+                                sep = sep_set
+                        elif sep_def > 0 and sep_set == -1:
+                                sep = sep_def
+                        else:
+                                sep = min(sep_set, sep_def)
+
```

Comment:
Ok, but then indentation looks wrong, the code block must not end up being part of the if block (especially after the continue).

Reply:
Sorry again. My editor doesn't like tabs. I fixed this.